app/views/shared/_errors.html.erb

```diff
@@ -0,0 +1,11 @@
+<% if obj.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(obj.errors.count, "error") %> prohibited this screening from being saved:</h2>
```

Comment:
I see the usage of this in few places for halls, movies, reservations, screening

WDYT about :arrow_down: ?
```suggestion
      <h2><%= pluralize(obj.errors.count, "error") %> prohibited this object from being saved:</h2>
```